enhancements/network/cloud-egress-ip.md

```diff
@@ -0,0 +1,445 @@
+---
+title: TBD - for now: Cloud API component for egress IP
+authors:
+  - "@alexanderConstantinescu"
+reviewers:
+  - "@danwinship"
+  - "@abhat"
+  - "@squeed"
+  - "@tssurya"
+  - "@trozet"
+approvers:
+  - TBD
+creation-date: 2020-12-16
+last-updated: -
+status: implementable
+---
+
+# TBD - for now: Cloud API component for egress IP
+
+This covers adding cloud provider support for the OpenShift networking feature:
+egress IP. It is not clear today what the extent of the component presented here
+will be, see: [Summary](#summary). I am thus leaving the title open-ended for
+further refinement once people review this enhancement proposal.
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+OVN-Kubernetes and openshift-sdn today support the feature egress IP on
```

Comment:
Can we try to summarize what EgressIP as a feature does?

Reply:
Besides what is mentioned in `Motivation`?